inspector/src/components/actionTabs/tabs/toolsTabComponent.tsx

```diff
@@ -382,7 +393,16 @@ export class ToolsTabComponent extends PaneComponent {
                             <ButtonLineComponent label="Export to Babylon" onClick={() => this.exportBabylon()} />
                             {
                                 !scene.getEngine().premultipliedAlpha && scene.environmentTexture && scene.environmentTexture._prefiltered && scene.activeCamera &&

```

Comment:
I didn't want to touch this line (yet), but wonder why this `scene.environmentTexture._prefiltered` is needed?

With this still in place, I can e.g. drop a gltf that already has an .env environment into the sandbox, and the "Generate .env texture" show up. Without that check, I can easily export the .env without problems. In this case I think it does make sense, as it allows you to convert an existing PNG-based .env to the new WebP format!

TL;dr I'd like to change this to:

```suggestion
                                !scene.getEngine().premultipliedAlpha && scene.environmentTexture && scene.activeCamera &&
```

Any reason not to do so?

Reply:
We should not be able to generate a texture from a none prefiltered cubemap, basically if people are relying on a lightprobe and such and use real time filtering, it is not smthg we should convert in an env file.